hathor/transaction/storage/transaction_storage.py

```diff
@@ -988,6 +998,58 @@ def _topological_fast(self) -> Iterator[BaseTransaction]:
         yield from cur_blocks
         yield from cur_txs
 
+    def _topological_fast2(self) -> Iterator[BaseTransaction]:
+        import heapq
+        from dataclasses import dataclass, field
+        cur_timestamp = settings.GENESIS_TIMESTAMP
+
+        @dataclass(order=True)
+        class Item:
+            timestamp: int
+            is_transaction: bool
```

Comment:
We did it on the other topological sort method because it used a DFS. I feel we don't need this here.

Reply:
Do you mean that we wouldn't need to prioritize blocks over transactions when they have the same timestamp? It doesn't cause any harm (it should have nearly undetectable performance downside), so I just did it. But I still think there could be a case where a transaction with the same timestamp as the block would need the block to add enough height to the best chain so the reward lock verification passes, so I'd rather just do it anyway.

Reply:
I agree. Thanks!